forc-pkg/src/pkg.rs

```diff
@@ -346,6 +346,16 @@ impl MemberFilter {
         }
     }
 
+    /// Returns a new `BuildFilter`, that only builds predicates.
```

Comment:
```suggestion
    /// Returns a new `MemberFilter`, that only builds predicates.
```

Should this be `MemberFilter` instead? Same for the other methods found above this.